Content.Server/Cuffs/CuffableSystem.cs

```diff
@@ -39,5 +41,14 @@ private void OnCuffableGetState(EntityUid uid, CuffableComponent component, ref
             // the iconstate is formatted as blah-2, blah-4, blah-6, etc.
             // the number corresponds to how many hands are cuffed.
         }
+
+        private void OnBuckleAttemptEvent(EntityUid uid, CuffableComponent component, ref BuckleAttemptEvent args)
+        {
+            // one hand cuffed is able to unbuckle
+            if (component.CuffedHandCount == 2)
```

Comment:
This breaks if the entity has more than 2 hands. Maybe it would be better to blanket deny it if there are any cuffs on

Reply:
This is intended as handcuffs are applied to only 2 hands, so if an entity has more than 2 hands it should be able to unbuckle.

But if it's better to just check for `CuffedHandCount > 0` I can change that, what do you think?

Reply:
It should be checking if all hands are cuffed. So if you have a free hand you can unbuckle.

Reply:
Not sure if we have a system to check that. Other parts do it like this:

```csharp
if (entMan.TryGetComponent<CuffableComponent>(_mind.OwnedEntity, out var cuffed)
                    && cuffed.CuffedHandCount > 0)
                    // You're not escaping if you're restrained!
                    agentIsEscaping = false;
```

I think I will just check that for now, until we implement body parts.

Reply:
> Not sure if we have a system to check that. Other parts do it like this:

you can check` if (cuffed.CuffedHandCount == handsComponent.Count)` or something (would just need to get the entity's handscomponent)